homeassistant/components/alexa/const.py

```diff
@@ -11,6 +11,21 @@
 CONF_FILTER = 'filter'
 CONF_ENTITY_CONFIG = 'entity_config'
 
+# Smart home constants
```

Comment:
Please move them back. There is no need in moving things to a constant file if they are only used in a single file.

Reply:
I structured this after the google_assistant component's use of constants but I will move them back.